web_src/js/modules/tippy.js

```diff
@@ -27,7 +27,7 @@ export function createTippy(target, opts = {}) {
       visibleInstances.add(instance);
     },
     arrow: `<svg width="16" height="7"><path d="m0 7 8-7 8 7Z" class="tippy-svg-arrow-outer"/><path d="m0 8 8-7 8 7Z" class="tippy-svg-arrow-inner"/></svg>`,
-    ...(opts?.role && {theme: opts.role}),
+    role: 'menu',
```

Comment:
A number of tippys will not have any `theme` set any more. I did like `theme` falling back to `role` when unspecified. Why remove this fallback?

Reply:
I see your point. I will revert this.

Reply:
But we need to set default role to "menu". Otherwise, some `createTippy` callers would forget to set the role again, then new could will conflict with the "auto hide tooltip" behavior, and such bug is difficult to find.

What's your opinion?

Reply:
Default both role and theme to `menu`, tooltips will all go through to tooltip init so get the correct values there, and anything else ought be be a "menu" of some kind.

Reply:
OK, then I think it's done now. Thank you

Reply:
I will push those defaults

Reply:
I think I have done it in d7a11f2e34dd87dd8ee061ce722af83325c24241 (previous ebf31b271dd77156694f6f1902b30ef0d5af4b2e before your force push)

Reply:
All tooltip inits must go through `attachTooltip`, so it's fine to have the menu default for both values.